### PR TITLE
feat(args): print multiple errors in github

### PR DESCRIPTION
**Pull request by i-keliukh**
Moved read_and_check_multiline_option function from utils to
HasErrorState. Improved error messages text.

### CODE REVIEW

**Comment by i-keliukh** [comment]
Relates to #261.

File: tests/test_multiline_variables.py [hunk 1]
```diff
@@ -3,47 +3,51 @@
...
 
 parser = ModuleArgumentParser()
-parser.add_argument('--argument', '-a', metavar='ARGUMENT')
+parser.add_argument('--argument', '-a', dest='HasErrorState.argument', metavar='ARGUMENT')
 
 namespace = parser.parse_args()
-print(read_and_check_multiline_option(namespace, 'argument', "{error_message}"))
+module = construct_component(HasErrorState, namespace)
+print(module.read_and_check_multiline_option('argument', "{error_message}"))
+if module.is_in_error_state():
```
Comment (k-dovgan) [inline tests/test_multiline_variables.py:24]: Should we, maybe, also check success scenarios?

File: tests/test_multiline_variables.py [hunk 2]
```diff
@@ -3,47 +3,51 @@
...
+    print(module.global_error_state.get_errors()[0])
 """
 
 
-@pytest.fixture()
-def parser():
-    parser = ModuleArgumentParser()
-    parser.add_argument('--argument', '-a', metavar='ARGUMENT')
-    yield parser
+def construct_test_module(args):
```
Comment (k-dovgan) [inline tests/test_multiline_variables.py:29]: Why is function better than fixture?

File: tests/test_multiline_variables.py [hunk 3]
```diff
@@ -66,6 +70,4 @@ def test_multiline_variable_stdin(tmp_path):
     del env['ARGUMENT']
 
     result = subprocess.run(["python3.7", script_path, "-a", "-"], **common_args, input="", check=False)
-    assert result.returncode != 0
-    assert "IncorrectParameterError" in result.stderr
-    assert error_message in result.stderr
+    assert error_message in result.stdout
```
Comment (k-dovgan) [inline tests/test_multiline_variables.py:73]: Why have we removed error type check?

File: tests/test_multiline_variables.py (hunk 1 above)
Comment (i-keliukh) [reply tests/test_multiline_variables.py:24]: All tests in this file check error and success scenarios.

File: tests/test_multiline_variables.py (hunk 2 above)
Comment (i-keliukh) [reply tests/test_multiline_variables.py:29]: Fixture is only needed when there is something to tear down.
Function is easier to navigate and debug in IDE .

File: tests/test_multiline_variables.py (hunk 3 above)
Comment (i-keliukh) [reply tests/test_multiline_variables.py:73]: Previously run of a test script result in an exception.
The script is changed to print error explicitly and it doesn't raise the exception.

**Review by k-dovgan — APPROVED** [review]

**Pull request merged**